specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2024-10-01-preview/definitions.json

```diff
@@ -9232,13 +9232,71 @@
         "description": {
           "type": "string",
           "description": "Description of the workspace."
+        },
+        "kind": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The kind of Gateway.",
+          "enum": [
+            "KubernetesManaged",
+            "Default"
+          ],
+          "x-ms-enum": {
+            "name": "GatewayKind",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "KubernetesManaged",
+                "description": "Gateway managed by Kubernetes resources."
+              },
+              {
+                "value": "Default",
+                "description": "Default kind of gateway."
+              }
+            ]
+          }
+        },
+        "provisioningState": {
```

Comment:
We can drop this, I forgot to update my design proposal

Reply:
Done.